examples/count_resources.py

```diff
@@ -0,0 +1,79 @@
+import requests
```

Comment:
add a header explaining what this does, and why you're using request instead of elabapi_python